Cargo.toml

```diff
@@ -17,5 +17,6 @@ members = [
     #"src/window/metal",
     #"src/window/sdl",
     #"examples",
-    "examples/core/quad"
+    "examples/core/quad",
+    "examples/render/quad_render",
```

Comment:
why not converting the existing triangle example (written against `gfx`) instead of adding a new one?

Reply:
I would have to rewrite the whole PSO stuff using `core` until it's ready in `render`, so I'd rather not. But I only plan to keep this new example temporarily. Once `render` will be more complete we can remove it and rewrite the old ones.